themes/dark-color-theme.json

```diff
@@ -1,73 +0,0 @@
-{
```

Comment:
Do we?  We don't supply any themes, which is a whole different category of extension.  Setting of individual syntax colors can be done with a bit more finesse.

Reply:
hmm we currently import these which we use in the Terrastruct app, I think they are here because it is a vscode theme, but perhaps we should move it out of this repo. We originally had themes here: https://github.com/terrastruct/d2-vscode/blob/5ede180a9221366b93df28d29546cc58b4c79e9a/package.json#L53-L64 . 

thoughts @terrastruct/engineering ?